### ISSUE TITLE
Problem building ramp on armv7

### ISSUE THREAD

**Issue by petevine** [Aatch/ramp#29]
Trying to build ramp (through pumpkin) on a 32-bit arm platform ends in this:

````
Compiling ramp v0.1.9
     Running `rustc /home/odroid/.cargo/registry/src/github.com-121aea75f9ef2ce2/ramp-0.1.9/src/lib.rs --crate-name ramp --crate-type lib -C opt-level=3 -C metadata=014f6b64cb3efd27 -C extra-filename=-014f6b64cb3efd27 --out-dir /tmp/pumpkin-master/target/release/deps --emit=dep-info,link -L dependency=/tmp/pumpkin-master/target/release/deps -L dependency=/tmp/pumpkin-master/target/release/deps --extern rand=/tmp/pumpkin-master/target/release/deps/librand-a5a71c849766362c.rlib --cap-lints allow`
/home/odroid/.cargo/registry/src/github.com-121aea75f9ef2ce2/ramp-0.1.9/src/ll/limb.rs:590:24: 590:39 error: the trait `core::ops::Sub<bool>` is not implemented for the type `ll::limb::Limb` [E0277]
/home/odroid/.cargo/registry/src/github.com-121aea75f9ef2ce2/ramp-0.1.9/src/ll/limb.rs:590             let high = ah - bh - carry;
                                                                                                                  ^~~~~~~~~~~~~~~
/home/odroid/.cargo/registry/src/github.com-121aea75f9ef2ce2/ramp-0.1.9/src/ll/limb.rs:555:5: 594:6 note: in this expansion of if_cfg! (defined in /home/odroid/.cargo/registry/src/github.com-121aea75f9ef2ce2/ramp-0.1.9/src/ll/limb.rs)
/home/odroid/.cargo/registry/src/github.com-121aea75f9ef2ce2/ramp-0.1.9/src/ll/limb.rs:590:24: 590:39 help: run `rustc --explain E0277` to see a detailed explanation
error: aborting due to previous error
Could not compile `ramp`.

Caused by:
  Process didn't exit successfully: `rustc /home/odroid/.cargo/registry/src/github.com-121aea75f9ef2ce2/ramp-0.1.9/src/lib.rs --crate-name ramp --crate-type lib -C opt-level=3 -C metadata=014f6b64cb3efd27 -C extra-filename=-014f6b64cb3efd27 --out-dir /tmp/pumpkin-master/target/release/deps --emit=dep-info,link -L dependency=/tmp/pumpkin-master/target/release/deps -L dependency=/tmp/pumpkin-master/target/release/deps --extern rand=/tmp/pumpkin-master/target/release/deps/librand-a5a71c849766362c.rlib --cap-lints allow` (exit code: 101)
````

The problem is non-existent on 32-bit x86.

**Comment by petevine** [comment]
Nope, still the same in ramp-0.1.12 using cargo - the master source builds fine though.

**Comment by huonw** [comment]
The minor version number was bumped, so getting the latest fixes requires using 0.2.0.

**Comment by petevine** [comment]
Aha! - the pumpkin project had 0.1.* in Cargo.toml which is why cargo update didn't work. Thx.
